rubocop-performance.gemspec

```diff
@@ -30,5 +30,6 @@ Gem::Specification.new do |s|
   }
 
   s.add_runtime_dependency('rubocop', '>= 0.87.0')
+  s.add_runtime_dependency('rubocop-ast', '>= 0.4.0')
```

Comment:
This was necessary to fix `Set`s in ruby 2.4. Please see https://github.com/rubocop-hq/rubocop-ast/issues/107 for more details.